.github/ISSUE_TEMPLATE/patternfly-issue-template.md

```diff
@@ -0,0 +1,17 @@
+---
+about: Please populate the PatternFly issue template so we can learn more about your
+  needs.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Describe the issue. What is the expected and unexpected behavior?
+
+### Please provide the steps to reproduce so we can better understand and address the issue. Feel free to link CodePen or another tool.
```

Comment:
Would be good to provide a ready-made environment that already pulls in PF etc

Reply:
This is on my to-do list, especially with the -org refactor. You can add a followup issue and assign me!